hvac/api/auth_methods/legacy_mfa.py

```diff
@@ -0,0 +1,172 @@
+
+        Supported methods:
+            POST: /auth/{mount_point}/mfa_config. Produces: 204 (empty body)
+
+        :param mount_point: The "path" the method/backend was mounted on.
+        :type mount_point: str | unicode
+        :param mfa_type: Enables MFA with given backend (available: duo)
+        :type mfa_type: str | unicode
+        :param force: If `True`, make the `mfa_config` request regardless of circumstance. If `False` (the default), verify
+            the provided `mount_point` is available and one of the types of methods supported by this feature.
+        :type force: bool
+        :return: The response of the configure MFA request.
+        :rtype: requests.Response
+        """
+        if mfa_type != "duo" and not force:
+            # The situation described via this exception is not likely to change in the future.
+            # However we provided that flexibility here just in case.
+            error_msg = 'Unsupported mfa_type argument provided "{arg}", supported types: "{mfa_types}"'
+            raise exceptions.ParamValidationError(
+                error_msg.format(
+                    mfa_types=",".join(SUPPORTED_MFA_TYPES),
+                    arg=mfa_type,
+                )
+            )
+        params = {
+            "type": mfa_type,
+        }
+
+        api_path = utils.format_url(
+            "/v1/auth/{mount_point}/mfa_config", mount_point=mount_point
+        )
+        return self._adapter.post(
+            url=api_path,
+            json=params,
+        )
+
+    def read_configuration(self, mount_point):
+        """Read the MFA configuration.
+
+        Supported methods:
+            GET: /auth/{mount_point}/mfa_config. Produces: 200 application/json
+
+
+        :param mount_point: The "path" the method/backend was mounted on.
+        :type mount_point: str | unicode
+        :return: The JSON response of the read_configuration request.
+        :rtype: dict
+        """
+        api_path = utils.format_url(
+            "/v1/auth/{mount_point}/mfa_config",
+            mount_point=mount_point,
+        )
+        return self._adapter.get(url=api_path)
+
+    def configure_duo_access(self, mount_point, host, integration_key, secret_key):
+        """Configure the access keys and host for Duo API connections.
+
+        To authenticate users with Duo, the backend needs to know what host to connect to and must authenticate with an
+        integration key and secret key. This endpoint is used to configure that information.
+
+        Supported methods:
+            POST: /auth/{mount_point}/duo/access. Produces: 204 (empty body)
+
+        :param mount_point: The "path" the method/backend was mounted on.
+        :type mount_point: str | unicode
+        :param host: Duo API host
+        :type host: str | unicode
+        :param integration_key: Duo integration key
+        :type integration_key: Duo secret key
+        :param secret_key: The "path" the method/backend was mounted on.
```

Comment:
There was another line, I missed it the first time

```suggestion
        :type integration_key: str | unicode
        :param secret_key: Duo secret key
```